src/hooks/words/use-words.hook.ts

```diff
@@ -20,7 +20,7 @@ export const useWords = (): UseWordIds => {
   const handleApiError = useApiErrorHook()
   const handleApplySemesterDetails = useApplySemesterDetails()
 
-  const handleRefresh: HandleRefresh = useRecoilCallback(
+  const onWords: HandleRefresh = useRecoilCallback(
```

Comment:
onGetWords

Reply:
Handled in [d98b17b](https://github.com/ajktown/wordnote/pull/158/commits/d98b17b1c2670af41b7b0116430834246b37fd00)